pycc/ccwfn.py

```diff
@@ -106,6 +106,13 @@ def __init__(self, scf_wfn, **kwargs):
             raise Exception("%s is not an allowed initial t2 amplitudes." % (it2_opt))
         self.it2_opt = it2_opt
 
+        valid_filter = [True,False]
```

Comment:
I'm not sure what's happening here.  Is this just a boolean test for filtering the residuals/amplitudes in the local simulation code?

Reply:
Yes, this is just a boolean test to access and use the local filter.

Reply:
Will it ever be anything other than boolean?  If not, then this seems like overkill.

Reply:
It does seem overkill. I thought it was needed to differentiate between the simulation code and the full-production local code since the `local` input only specifies what local approximation to use. Looking at line 151-156, the simulation code (when `filter` is `True`) goes through the local object to obtain the transformation matrices needed to filter out nonlocal components during the residual update. If its the full-production (when `filter` is `False`) then it also goes to the local object to obtain the local representation of the integrals and overlap terms. Unless there is a different way.